sites/website/src/docs/design-systems/overview.md

```diff
@@ -0,0 +1,64 @@
+---
+id: overview
+title: What is the Design System?
+sidebar_label: Overview
+custom_edit_url: https://github.com/microsoft/fast/edit/master/sites/website/src/docs/design-systems/overview.md
+---
+A *Design System* is somewhat of an amorphous term but can generally be thought of as a collection of resources for interactive media that promotes brand alignment. While that definition is intentionally broad, in UI development, Design Systems generally manifest as component libraries surrounded by usage guidance and design principles.
```

Comment:
What about... "A *Design System* is a collection of resources for interactive media that promotes brand alignment. While that definition is intentionally broad, in UI development, Design Systems manifest as component libraries surrounded by usage guidance and design principles."

Reply:
The point is that there is a very broad definition in the industry about what a design system actually is - I'm not sure we want to declare the industry term here, just set the stage for how the term is used in FAST today